src/core/i18n/utils.js

```diff
@@ -242,7 +242,8 @@ export function makeI18n(i18nData: I18nConfig, lang: string, _Jed: Jed = Jed) {
 
   // TODO: move all of this to an I18n class that extends Jed so that we
   // can type-check all the components that rely on the i18n object.
-  i18n.formatNumber = (number) => number.toLocaleString(lang);
+  // Note: the available locales for tests are controlled in tests/setup.js
+  i18n.formatNumber = (number) => new Intl.NumberFormat(lang).format(number);
```

Comment:
Browser support for this should be good enough. The change was necessary since it's a more universal approach and works for testing too.